pkg/sidecar/config/config.go

```diff
@@ -296,6 +301,15 @@ func (s *ScyllaConfig) validateCpuSet(ctx context.Context, cpusAllowed string, s
 	return nil
 }
 
+func (s *ScyllaConfig) setupIOTuneCache() error {
```

Comment:
I don't see the object being used, maybe it should be just a function

Reply:
done